### PR TITLE
Add information about feeds

### LINKED ISSUE #5: Remonter plus d'informations sur la page d'un jeu de données

### PR DESCRIPTION
**Pull request by l-vincent-l**
New feed page :
feed_info

Closes #5

Depends on etalab/datatools-server#3

### CODE REVIEW

**Comment by bonjourmauko** [comment]
It could be nice to add a screenshot for UI changes.

**Comment by l-vincent-l** [comment]
fixes #21 fixes #22 fixes #6

File: lib/public/components/PublicFeedSourceViewer.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ export default class PublicFeedSourceViewer extends Component {
               ? <ActiveFeedVersionNavigator
                 feedSource={fs}
                 routeParams={this.props.routeParams}
+                user={this.props.user}
```
Comment (bonjourmauko) [inline lib/public/components/PublicFeedSourceViewer.js]: Do we really need it?

Comment (l-vincent-l) [reply lib/public/components/PublicFeedSourceViewer.js]: Yep, try to remove it.

File: lib/manager/components/version/FeedVersionDetails.js [hunk 2]
```diff
@@ -32,7 +32,9 @@ export default class FeedVersionDetails extends Component {
   render () {
     const {user, validationJob, version} = this.props
     const isPublished = version.id === version.feedSource.publishedVersionId
-    const userCanManageFeed = user.permissions.hasFeedPermission(version.feedSource.organizationId, version.feedSource.projectId, version.feedSource.id, 'manage-feed')
+    const userCanManageFeed = user && user.permissions &&
```
Comment (bonjourmauko) [inline lib/manager/components/version/FeedVersionDetails.js:35]: Works fine 😄 . Alternative syntax :

```javascript
const userCanManageFeed = (organizationId, projectId, feedSourceId) => {
  if(!user) { return false; }
  if(!user.permissions) { return false; }
  return user.permissions.hasFeedPermission(organizationId, projectId, feedSourceId, 'manage-feed');
}(version.feedSource.organizationId, version.feedSource.projectId, version.feedSource.id);
```

**Comment by bonjourmauko** [comment]
Rebased.

**Review by bonjourmauko — CHANGES_REQUESTED** [review]
Doesn't work when not logged in:

capture d ecran 2017-07-20 a 17 12 44

**Review by bonjourmauko — DISMISSED** [review]

**Comment by bonjourmauko** [comment]
Rebase + fixup

**Review by bonjourmauko — APPROVED** [review]

**Pull request merged**